trojsten/rules/kms.py

```diff
@@ -18,6 +23,7 @@
 KMS_ALFA = "alfa"
 KMS_BETA = "beta"
 
+KMS_POINTS_FOR_SUCCESSFUL_SEMESTER = 4
```

Comment:
```suggestion
KMS_POINTS_FOR_SUCCESSFUL_SEMESTER = 90
```
Nemalo by tu byť číslo 90 ako počet bodov potrebných na úspešný semester?

Reply:
Áno, to je pozostatok môjho testovania.